src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationHandler.cs

```diff
@@ -159,14 +160,44 @@ protected override AuthenticationTicket AuthenticateCore()
             return new TokenResponse(oauth2Token);
         }
 
-        protected virtual async Task<AuthenticationTicket> GetUserInformationAsync(AuthenticationProperties properties, TokenResponse tokens)
+        protected virtual async Task<AuthenticationTicket> CreateTicketAsync(AuthenticationProperties properties, TokenResponse tokens)
```

Comment:
Not a "necessary" change, but I believe that `CreateTicketAsync` is a far more appropriate name for this method.